web/test/support/index.js

```diff
@@ -1,6 +1,8 @@
 import './commands';
 
 before(() => {
+  cy.task('db:delete', {collection: 'projects', subCollection: 'pings'})
+  cy.task('db:delete', {collection: 'repositories', subCollection: 'statuses'})
```

Comment:
It might be cleaner if the `collection` and `subcollections` (array of multiples) where all in a single object, and all got deleted, what do you think?

```typescript
{
    collection: 'repositories', 
    subCollections: [ 'statuses' ]
}
```